src/core/cubic.c

```diff
@@ -573,8 +573,8 @@ CubicCongestionControlOnDataLost(
             LossEvent->PersistentCongestion);
     }
 
-    CXPLAT_DBG_ASSERT(Cubic->BytesInFlight >= LossEvent->NumRetransmittableBytes);
-    Cubic->BytesInFlight -= LossEvent->NumRetransmittableBytes;
+    CXPLAT_DBG_ASSERT(Cubic->BytesInFlight >= LossEvent->NumLostRetransmittableBytes);
+    Cubic->BytesInFlight -= LossEvent->NumLostRetransmittableBytes;
```

Comment:
Is this correct? You changed the `LossEvent` struct by replacing `NumRetransmittableBytes` with `NumAckedRetransmittableBytes` **and** `NumLostRetransmittableBytes`. Does that mean they add up to the original? If so, should this be using the sum of those two?

Reply:
> Does that mean they add up to the original?
No. 
`NumLostRetransmittableBytes` is the number of lost bytes and `NumAckedRetransmittableBytes` is number of acked bytes. You can find it in function `QuicLossDetectionDetectAndHandleLostPackets` from `loss_detection.c`.
In Cubic, we only need to know the number of lost bytes. But we need to use the acked bytes of updating the recovery window in BBR. That's why for the change.